types/linkifyjs/linkifyjs-tests.ts

```diff
@@ -1,141 +1,163 @@
+        formatHref(href, type) {
+            if (type === "hashtag") {
+                href = "https://twitter.com/hashtag/" + href.substring(1);
+            }
+            return href;
         }
-        return href;
-    }
-});
-
-linkifyHtml("Hey @dhh, check out issue #23", {
-    formatHref: {
-        mention(href) {
-            return "https://github.com" + href;
-        },
-        ticket(href) {
-            return (
-                "https://github.com/SoapBox/linkifyjs/issues/" +
-                href.substring(1)
-            );
+    });
+
+    linkifyHtml("Hey @dhh, check out issue #23", {
+        formatHref: {
+            mention(href) {
+                return "https://github.com" + href;
+            },
+            ticket(href) {
+                return (
+                    "https://github.com/SoapBox/linkifyjs/issues/" +
+                    href.substring(1)
+                );
+            }
         }
-    }
-});
+    });
 
-/* ignoreTags */
+    /* ignoreTags */
 
-linkifyHtml(
-    // tslint:disable-next-line:prefer-template
-    'Please ignore <script>var a = {}; a.com = "Hi";</script> \n' +
-        "but do <span>b.ca</span>",
-    {
-        ignoreTags: ["script", "style"]
-    }
-);
+    linkifyHtml(
+        // tslint:disable-next-line:prefer-template
+        'Please ignore <script>var a = {}; a.com = "Hi";</script> \n' +
+            "but do <span>b.ca</span>",
+        {
+            ignoreTags: ["script", "style"]
+        }
+    );
 
-/* nl2br */
+    /* nl2br */
 
-/* tagName */
+    /* tagName */
 
-linkifyHtml("github.com", {
-    tagName: "span"
-});
+    linkifyHtml("github.com", {
+        tagName: "span"
+    });
 
-linkifyHtml("#swag", {
-    tagName: {
-        hashtag: "span"
-    }
-});
+    linkifyHtml("#swag", {
+        tagName: {
+            hashtag: "span"
+        }
+    });
 
-/* target */
+    /* target */
 
-linkifyHtml("github.com", {
-    target: "_parent"
-});
+    linkifyHtml("github.com", {
+        target: "_parent"
+    });
 
-linkifyHtml("test-email@example.com", {
-    target: {
-        url: "_parent",
-        email: null
-    }
-});
+    linkifyHtml("test-email@example.com", {
+        target: {
+            url: "_parent",
+            email: null
+        }
+    });
 
-/* validate */
+    /* validate */
 
-// Don't linkify links that don't begin in a protocol
-// e.g., "http://google.com" will be linkified, but "google.com" will not.
-linkifyHtml("www.google.com", {
-    validate: {
-        url(value) {
-            return /^(http|ftp)s?:\/\//.test(value);
+    // Don't linkify links that don't begin in a protocol
+    // e.g., "http://google.com" will be linkified, but "google.com" will not.
+    linkifyHtml("www.google.com", {
+        validate: {
+            url(value) {
+                return /^(http|ftp)s?:\/\//.test(value);
+            }
         }
-    }
+    });
+});
+
+describe("linkifyjs/react", () => {
+    it("should render a Linkify component", () => {
```

Comment:
Regarding using `describe` and `it`, do you know if the tests in this repo are actually run? I was under the assumption that they are not, and that these files are only type-checked. I'm not sure though

Reply:
I took the lead from the `react` declarations, I am assuming tests are actually run because mocha is a dependency of DefinitelyTyped.